<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Play">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始播放</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Audio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录音</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>总时长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AllTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余时长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hasTime">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>录音时长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AudioTime">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>3333</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Wav">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成WAV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Resample">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重采样</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_aacEncode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>aac编码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_aacDcode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>aac解码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showBMP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示BMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_customDraw">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>手动绘制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>录制视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showYUV">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示YUV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showYUVLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示YUV在Label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_playYuv">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放YUV视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_h264Encode">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>h264编码</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
